things/api/things/http/endpoint_test.go

```diff
@@ -142,15 +142,15 @@ func TestCreateThing(t *testing.T) {
 			req:         data,
 			contentType: contentType,
 			auth:        wrongValue,
-			status:      http.StatusForbidden,
+			status:      http.StatusUnauthorized,
```

Comment:
Please verify how this relates to https://github.com/mainflux/mainflux/pull/1140